recipes/reprounzip-qt/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "reprounzip-qt" %}
+{% set version = "1.0.11" %}
+{% set sha256 = "6738edae930aa8e447ec1c6e1fb0341bbe84ca94ed7e54e14c2abbb3b21af563" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}-1.tar.gz
```

Comment:
This will break when you update the version...you could add `{% set postversion = "-1" %}` and change this to `{{name }}-{{ version }}{{ postversion }}.tar.gz` or something.